errbot/backends/hipchat.py

```diff
@@ -590,7 +590,7 @@ def _hipchat_upload(self, stream):
                 log.error('Request text: %s.', resp.text)
                 stream.error()
         except Exception:
-            log.exception(f'Upload of {stream.name} to {stream.identifier.channelname} failed.')
```

Comment:
why f string is removed?

Reply:
I don't recall the specific reason but i'm sure it was because the linter complained about lazy logging.

I really appreciate the review!